### PR TITLE
CRM Common tests refactoring

### PR DESCRIPTION
**Pull request by olga-nesterova**

### CODE REVIEW

File: lib/teamlab/modules/crm/crm_common.rb [hunk 1]
```diff
@@ -23,20 +23,28 @@ def get_all_currency_rates
       @request.get(%w[currency rates], {})
     end
 
-    def set_currency_rate(from = 'EUR', to = 'USD', rate = '1.0')
+    def create_currency_rate(from = 'EUR', to = 'USD', rate = '1.0')
```
Comment (ShockwaveNN) [inline lib/teamlab/modules/crm/crm_common.rb:26]: Add entry to changelog that name of method changed

File: lib/teamlab/modules/crm/crm_common.rb [hunk 2]
```diff
@@ -23,20 +23,28 @@ def get_all_currency_rates
...
+
     def delete_currency_rate_by_id(id)
       @request.delete(['currency', 'rates', id.to_s], {})
     end
 
     def set_is_portal_configured(options = {})
       @request.put(%w[settings], options)
     end
+
+    def change_web_form_key
```
Comment (ShockwaveNN) [inline lib/teamlab/modules/crm/crm_common.rb:46]: Add basic description if possible for new method

File: lib/teamlab/modules/crm/crm_common.rb [hunk 3]
```diff
@@ -23,20 +23,28 @@ def get_all_currency_rates
...
-    def set_currency_rate(from = 'EUR', to = 'USD', rate = '1.0')
+    def create_currency_rate(from = 'EUR', to = 'USD', rate = '1.0')
       @request.post(%w[currency rates], fromCurrency: from, toCurrency: to, rate: rate)
     end
 
     def get_currency_rate_by_id(id)
       @request.get(['currency', 'rates', id.to_s], {})
     end
 
+    def get_currency_rate_by_currency(from_currency, to_currency)
```
Comment (ShockwaveNN) [inline lib/teamlab/modules/crm/crm_common.rb:34]: Add documentation

**Review by ShockwaveNN — CHANGES_REQUESTED** [review]

File: lib/teamlab/modules/crm/crm_common.rb (hunk 2 above)
Comment (olga-nesterova) [reply lib/teamlab/modules/crm/crm_common.rb]: Deleted method as there is no parameters in [documentation](https://api.onlyoffice.com/portals/method/crm/put/api/2.0/crm/settings/webformkey/change) for changing Website Contact Form key and no information what it is for

**Review by ShockwaveNN — APPROVED** [review]

**Review by ShockwaveNN — APPROVED** [review]

**Pull request merged**